lib/storage/bucket.js

```diff
@@ -288,8 +306,51 @@ Bucket.prototype.getMetadata = function(callback) {
 };
 
 /**
+ * Restrict access to your bucket's contents to only project owners.
+ *
+ * @param {function=} callback - The callback function.
+ *
+ * @example
+ * bucket.makePrivate(function(err) {});
+ */
+Bucket.prototype.makePrivate = function(callback) {
+  callback = callback || util.noop;
+
+  this.setMetadata({
+    predefinedAcl: 'private'
+  }, function(err) {
+    callback(err || null);
+  });
+};
+
+/**
+ * Make the files in your bucket publicly readable.
+ *
+ * @param {function=} callback - The callback function.
+ *
+ * @example
+ * bucket.makePublic(function(err) {});
+ */
+Bucket.prototype.makePublic = function(callback) {
+  callback = callback || util.noop;
+
+  this.setMetadata({
+    predefinedAcl: 'publicRead'
```

Comment:
Does this make all the files in the bucket public as well? ie,

- create bucket
- create file in bucket
- `file.makePrivate()`
- `bucket.makePublic()`

Is file public? (I argue it should be.)